examples/restricted-selectable/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang='en'>
+
+<head>
+  <meta charset="utf-8">
+
+  <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+  <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap-theme.min.css">
+  <link href="//veg.github.io/phylotree.js/phylotree.css" rel="stylesheet">
+
+  <script src="//code.jquery.com/jquery.js"></script>
+  <script src="//netdna.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+  <script src="//d3js.org/d3.v3.min.js"></script>
+  <script src="/phylotree.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js" charset="utf-8"></script>
+</head>
+
+<body>
+
+  <button id="select_all">Select All</button>
+  <button id="select_none">Select None</button>
+  <button id="select_leaf">Select Leaf Nodes</button>
+  <button id="select_internal">Select Internal Nodes</button>
+
+  <svg id="tree_display"></svg>
+
+  <script>
+    d3.text("yokoyama.nwk", function(error, newick) {
+      var height = 400,
+        width = 400,
+
+        tree = d3.layout.phylotree()
+        .svg(d3.select("#tree_display"))
+        .options({
+          'left-right-spacing': 'fit-to-size',
+          // fit to given size top-to-bottom
+          'top-bottom-spacing': 'fit-to-size',
+          // fit to given size left-to-right
+          'selectable': false,
```

Comment:
Enabling `restricted-selectable` below seems to disable the default branch selection features. If this is so, having `selectable: false` here seems redundant.